bin/setup_controller

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e
+
+setup_base=
+faucet_port=6653
+prom_port=9302
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        -s|--setup_base)
+            skip_docker_build=y
```

Comment:
this doesn't seem right... setup_base is always empty, and this flag then doesn't build docker?

Reply:
Done